tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+description = Default tox environments list
+envlist =
+    style,{py37,py38,py39,py310}{,-coverage},doc
```

Comment:
```suggestion
    style,{py37,py38,py39,py310},{-coverage},doc
```

??

Reply:
This was the syntax used in the pygeometry repository but I can change it! What do you think @jorgepiloto ?

Reply:
Yes, agreed, this is totally fine

Reply:
Fair. I didnt know that is why I asked. :)